README.md

```diff
@@ -166,4 +209,4 @@ Because some compilers don't support the `typeof` operator, which is used for st
 | erase `4` items from `vec` at index `3` | `vector_erase(vec, type, 3, 4);`                 | no (moves elements)     |
 | remove item at index `3` from `vec`     | `vector_remove(vec, type, 3);`                   | no (moves elements)     |
 | add `item` to the vector `vec`          | `type* temp = vector_add_dst(&vec, type);`       | yes                     |
-| insert `item` into `vec` at index `9`   | `type* temp = vector_insert_dst(&vec, type, 9);` | yes                     |
\ No newline at end of file
+| insert `item` into `vec` at index `9`   | `type* temp = vector_insert_dst(&vec, type, 9);` | yes                     |
```

Comment:
It seems you added or removed a trailing blank line. I don't think this is the desired change.

Reply:
Take another look at the full diff, please. I undid all the formatting changes you mentioned.